test/solver_tests/maxcut_test.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+import networkx as nx
+import pandas as pd
+
+from cassiopeia.solver.MaxCutSolver import MaxCutSolver
+from cassiopeia.solver import graph_utilities
+from cassiopeia.solver import solver_utilities
+
+class MaxCutSolverTest(unittest.TestCase):
```

Comment:
You should add some more tests here around the helper function `check_if_cut`. Also I know that this algorithm is stochastic, but there must be a very simple case that it's guaranteed to find the right tree with (e.g. PP?). You should write a test making sure it passes this simple case.